cvat/settings/base.py

```diff
@@ -264,15 +265,40 @@ def add_ssh_keys():
     "USER_SANDBOX_PROJECTS": 3,
     "TASKS_IN_USER_SANDBOX_PROJECT": 5,
     "USER_OWNED_ORGS": 1,
-    "USER_SANDBOX_CLOUD_STORAGES": 10,
+    "USER_SANDBOX_CLOUD_STORAGES": 1,
+    "USER_SANDBOX_PROJECT_WEBHOOKS": 10,
+    "USER_SANDBOX_LAMBDA_CALL_OFFLINE": 0,
 
     "ORG_TASKS": 10,
     "ORG_PROJECTS": 3,
+    "ORG_MEMBERS": 3,
     "TASKS_IN_ORG_PROJECT": 5,
-    "ORG_CLOUD_STORAGES": 10,
-    "ORG_COMMON_WEBHOOKS": 20,
+    "ORG_CLOUD_STORAGES": 1,
+    "ORG_COMMON_WEBHOOKS": 10,
+    "ORG_PROJECT_WEBHOOKS": 10,
+    "ORG_LAMBDA_CALL_OFFLINE": 0,
+}
+
+
+ORG_LIMITS_MAPPING = {
```

Comment:
@kirill-sizov , could you please explain why we need the mapping in settings?

Reply:
We have limit names that used by iam, and we also have limit names that we display to user.
And so since we have two different names for each limit we need to define mapping between them.